GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/DisableOpacityMaskOnEndScroll.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Interactivity;
+using System.Windows.Media;
+
+namespace GalaxyZooTouchTable.Behaviors
+{
+    public class DisableOpacityMaskOnEndScroll : Behavior<ScrollViewer>
+    {
+        private LinearGradientBrush InitialOpacityMask { get; set; }
+
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+
+            AssociatedObject.ScrollChanged += AssociatedObject_ScrollChanged;
+            SetOpacityMask();
+        }
+
+        private void SetOpacityMask()
+        {
+            var gradientCollection = new GradientStopCollection();
+            var gradientStop1 = new GradientStop(Color.FromRgb(0, 0, 0), 0);
+            var gradientStop2 = new GradientStop(Color.FromArgb(0, 0, 0, 0), 1);
+            gradientCollection.Add(gradientStop1);
+            gradientCollection.Add(gradientStop2);
+            
+
+            Point StartPoint = new Point();
+            Point EndPoint = new Point();
+            if (IsHorizontal)
+            {
+                StartPoint.X = StartPercent;
+                EndPoint.X = 1;
+            }
+            else
+            {
+                StartPoint.Y = StartPercent;
+                EndPoint.Y = 1;
+            }
+
+            InitialOpacityMask = new LinearGradientBrush(gradientCollection, StartPoint, EndPoint);
+            AssociatedObject.OpacityMask = InitialOpacityMask;
+        }
+
+        public static readonly DependencyProperty IsHorizontalProperty =
+            DependencyProperty.Register("IsHorizontal", typeof(Boolean), typeof(DisableOpacityMaskOnEndScroll));
+
+        public bool IsHorizontal
+        {
+            get { return (bool)GetValue(IsHorizontalProperty); }
+            set { SetValue(IsHorizontalProperty, value); }
+        }
+
+        public static readonly DependencyProperty StartPercentProperty =
+            DependencyProperty.Register("StartPercent", typeof(double), typeof(DisableOpacityMaskOnEndScroll));
+
+        public double StartPercent
```

Comment:
Should these properties be `public`?

Reply:
Yes, the getter and setter need to be `public` so the views can pass in a property, but the `static readonly` value above can be `private`.